.gitignore

```diff
@@ -2,6 +2,7 @@ generated/
 build/
 Reference/
 .vscode/
+subgraph.yaml
```

Comment:
Do you want this to ignore all `subgraph.yaml` files in the project or just at the root directory?

If the latter, I think it should be `**/subgraph.yaml`.

If that's the idea, can it happen that some protocol contains a single subgraph.yaml file without it being templatized? If so, do we want to ignore that one too?

Reply:
there would be no subgraph.yaml at the root directory?

There are 2 subgraphs that are not templateized yet, but could be. I just left the `subgraph.yaml` in there for those. we could "un-.gitignore" those `!subgraph.yaml` in their respective folders?

Reply:
Oh, never mind. I thought this way it was only going to ignore `subgraph.yaml` if placed at the root. My bad.

> There are 2 subgraphs that are not templateized yet, but could be. I just left the subgraph.yaml in there for those. we could "un-.gitignore" those !subgraph.yaml in their respective folders?

If they are already commited it will be fine. It's when we start a new one that we'll need to be explicit to not forget to commit those subgraph.yaml